meta-refkit-core/recipes-ostree/refkit-ostree/files/refkit-ostree/src/refkit-ostree.c

```diff
@@ -0,0 +1,1642 @@
+/*
+ * Copyright (c) 2017, Intel Corporation
```

Comment:
I have run static analysis locally on my machine on the latest code (with full defect detection enabled). It did not discover anything relevant.